### PR TITLE
Dswx ni beta point

### PR DESCRIPTION
**Pull request by hb6688**
Interface to detect operation mode for input NISAR RTC.

### CODE REVIEW

**Review by hb6688 — COMMENTED** [review]
I agree with the updates from Jungkyo,  LGTM.

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Then, from inside the cloned repository, build the Docker image:
 Load the Docker container image onto your computer:
 
 ```bash
-docker load -i docker/dockerimg_dswx_s1_final_1.0.tar
+docker load -i docker/dockerimg_dswx_s1_final_patch_1.1.tar
```
Comment (oberonia78) [inline README.md:66]: remove `README.md` in your PR.

File: build_docker.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 IMAGE=opera/dswx-s1
-tag=final_1.0
+tag=final_patch_1.1
```
Comment (oberonia78) [inline build_docker.sh:4]: remove `build_docker.sh`

**Review by oberonia78 — APPROVED** [review]
LGTM.

**Pull request merged**